src/testcafe-runner.ts

```diff
@@ -225,6 +229,11 @@ export function buildCommandLine(suite: Suite | undefined, projectPath: string,
   return cli;
 }
 
+function isCDPDisabled() {
```

Comment:
Would also be good to explain this. You are checking if CDP is disabled, but underneath you are checking whether native automation is disabled. One thing leads to another, but to someone who is not familiar with TestCafe, this would be arcane knowledge.

Reply:
For sure!